ENV.MD

```diff
@@ -0,0 +1,427 @@
+  - 1 => Enables fail2ban service
+
+If you enable Fail2Ban, don't forget to add the following lines to your `docker-compose.yml`:
+
+    cap_add:
+      - NET_ADMIN
+
+Otherwise, `iptables` won't be able to ban IPs.
+
+##### SMTP_ONLY
+
+  - **empty** => all daemons start
+  - 1 => only launch postfix smtp
+
+##### SSL_TYPE
+
+  - **empty** => SSL disabled
+  - letsencrypt => Enables Let's Encrypt certificates
+  - custom => Enables custom certificates
+  - manual => Let you manually specify locations of your SSL certificates for non-standard cases
+  - self-signed => Enables self-signed certificates
+  - _any other value_ => SSL required, settings by default
+
+Please read [the SSL page in the wiki](https://github.com/tomav/docker-mailserver/wiki/Configure-SSL) for more information.
+
+##### TLS_LEVEL
+
+  - **empty** => modern
+  - modern => Enables TLSv1.2 and modern ciphers only. (default)
+  - intermediate => Enables TLSv1, TLSv1.1 and TLSv1.2 and broad compatibility ciphers.
+  - old => NOT implemented. If you really need it, then customize the TLS ciphers overriding postfix and dovecot settings [ wiki](https://github.com/tomav/docker-mailserver/wiki/
+
+##### SPOOF_PROTECTION
+Configures the handling of creating mails with forged sender addresses.
+  - **empty** => Mail address spoofing allowed. Any logged in user may create email messages with a forged sender address. See also [Wikipedia](https://en.wikipedia.org/wiki/Email_spoofing)(not recommended, but default for backwards compatibility reasons)
+  - 1 => (recommended) Mail spoofing denied. Each user may only send with his own or his alias addresses. Addresses with [extension delimiters](http://www.postfix.org/postconf.5.html#recipient_delimiter) are not able to send messages.
+
+##### ENABLE_SRS
+Enables the Sender Rewriting Scheme. SRS is needed if your mail server acts as forwarder. See [postsrsd](https://github.com/roehling/postsrsd/blob/master/README.md#sender-rewriting-scheme-crash-course) for further explanation.
+  - **0** => Disabled
+  - 1 => Enabled
+
+##### PERMIT_DOCKER
+
+Set different options for mynetworks option (can be overwrite in postfix-main.cf)
+  - **empty** => localhost only
+  - host => Add docker host (ipv4 only)
+  - network => Add the docker default bridge network (172.16.0.0/12); **WARNING**: `docker-compose` might use others (e.g. 192.168.0.0/16) use `PERMIT_DOCKER=connected-networks` in this case
+  - connected-networks => Add all connected docker networks (ipv4 only)
+
+##### VIRUSMAILS_DELETE_DELAY
+
+Set how many days a virusmail will stay on the server before being deleted
+  - **empty** => 7 days
+
+
+##### ENABLE_POSTFIX_VIRTUAL_TRANSPORT
+
+This Option is activating the Usage of POSTFIX_DAGENT to specify a ltmp client different from default dovecot socket.
+
+- **empty** => disabled
+- 1 => enabled
+
+##### POSTFIX_DAGENT
+
+Enabled by ENABLE_POSTFIX_VIRTUAL_TRANSPORT. Specify the final delivery of postfix
+
+- **empty**: fail
```

Comment:
instead of failing the container on an undefined value we should stick to a sensible default. ´lmtp:unix:private...´ seems a good choice.